lib/ast-input-transform.js

```diff
@@ -104,15 +108,15 @@ class AngleBracketInputPolyfill {
 
     let visitor = {
       ElementNode(node) {
-        let tag = node.tag.toLowerCase();
```

Comment:
That's the culprit, which I copy pasted blindly from the link-to transform, which does not work here obviously 🙈
Sorry about that!